src/routes/users.js

```diff
@@ -459,6 +459,48 @@ router.post('/reset_pw/user', userValidation.sendEmail, forgetPassword);
     }
  */
 router.patch('/password/reset/:id/:token', userValidation.reset, resetPassword);
```

Comment:
hey @Moise1 , 
would you please not mixt the documentation with the routes? move your documentation up above this route... It would be less messy.

Reply:
@T2Wil  I've applied your feedback. Please take a look again.